netsuite/10.0/modules/ROOT/pages/netsuite-troubleshooting.adoc

```diff
@@ -82,6 +83,17 @@ To perform a search for Sales Order objects and other unsupported objects, you c
 * Develop a SuiteScript, deploy it as a RESTlet, and call it from the connector.
 * Create a saved search and call it from the connector.
 
+
+[[triggers-custom-records]]
+== Inability to Set a Custom Record as an Object Type for Triggers
+
+The connector does not support custom records as object types for all triggers (On Deleted Object, On Modified Object, and On New Object).
+
+You can do either of the following as an alternative:
+
+* Set CUSTOM_RECORD_BASIC and work with this type, however it is available only for On Deleted Object and On New Object.
+* Set up a *Search* operator that executes for a certain amount of time and works as a manually developed trigger.
```

Comment:
I don't know if people will know how to do this? You can ask Santiago if this needs further instructions. Not sure it would be common knowledge.